pandas/core/arrays/base.py

```diff
@@ -219,7 +217,7 @@ def _formatting_values(self):
         # type: () -> np.ndarray
         # At the moment, this has to be an array since we use result.dtype
         """An array of values to be printed in, e.g. the Series repr"""
-        raise AbstractMethodError(self)
+        raise np.array(self)
```

Comment:
Suggested by @jorisvandenbossche in #19268, this should be an OK default implementation.